clients/fides-js/src/lib/consent-utils.ts

```diff
@@ -0,0 +1,77 @@
+import {ConsentBannerOptions} from "~/lib/consent-types";
+
+
+// Adapted from https://gist.github.com/henrik/1688572?permalink_comment_id=4317520#gistcomment-4317520o
+// (NOTE: Surprisingly, there's not really a list of these anywhere easily...?)
+const EU_COUNTRY_CODES = [
```

Comment:
TBH I really don't know where this should live. See this discussion too... https://github.com/ethyca/fides/issues/3123#issuecomment-1528146093